### ISSUE TITLE
A11y_VSCode_Problems Panel_AI4D:Name of the filter edit box is provided as "No problems have been detected in the workspace so far" which is incorrect.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#97240]
**Environment Details:**
VSCode Version : 1.44.2
OS Version : Win10
Additional Details:
Screenreader: NVDA[2019.3.1], JAWS[2020.70]
GitHubTags: #A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCode; #MAS1.3.1; #MAS4.2.1; #A11ySev2; #AI4D

**Repro Steps:**
1)Open Visual studio code editor  
2)Press ctrl+shift+M
3)Validate it's possible to navigate between items in the problems view
4)Open Accessibility Insights for desktop and check the name of the filter edit box present besides to debug console tab after click on Problems Tab.
**Actual:**
Name of the filter edit box is provided as "No problems have been detected in the workspace so far" which is incorrect.
**Expected:**
Name should be provided as "Filter" and control type should be edit.

**User Impact:**
Screen reader user unable to get the purpose of the control.

**Attachment For Reference:**
![fiter edit box](https://user-images.githubusercontent.com/53940787/81397301-a4fbf200-9144-11ea-890f-1b5f8906559e.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
Assigning to @isidorn for insights.

**Comment by isidorn** [comment]
@sandy081 thanks for the ping. The issue is that you are setting `aria-labelledby=markers-panel-arialabel` on the filter input. Just remove that and it will be good since you set a proper aria-label on the element

So this line should be removed https://github.com/microsoft/vscode/blob/8f0f44cd0f718e9c2277758b400b7b34676d7bf6/src/vs/workbench/contrib/markers/browser/markersViewActions.ts#L326

![Screenshot 2020-05-11 at 15 25 09](https://user-images.githubusercontent.com/1926584/81566799-aecb6280-939b-11ea-952e-472937ebf863.png)

**Comment by sandy081** [comment]
Thanks.

**Comment by nirmalsahoo** [comment]
@sandy081 The issue is still reproduced that the name is provided as No problems have been detected in the workspace so far". The name should be Filter.

**Comment by nirmalsahoo** [comment]
Verified and Closed .Please refer attachment below
**
![image](https://user-images.githubusercontent.com/53940787/83132834-9a0efe80-a0ff-11ea-8c7a-9c3a1c11e91b.png)
**

**Comment by isidorn** [comment]
Thanks for verifiying, adding verified label.
